frontend/src/features/comparisons/Comparison.tsx

```diff
@@ -80,21 +103,25 @@ const Comparison = () => {
       if (searchParams.get(videoKey) !== videoId) {
         searchParams.delete(videoKey);
         if (videoId) {
-          searchParams.append(videoKey, videoId);
+          if (Object.values(uidParams).includes(videoKey)) {
+            searchParams.append(videoKey, UID_YT_NAMESPACE + videoId);
+          } else {
+            searchParams.append(videoKey, videoId);
+          }
```

Comment:
I think that we could find a simpler way.

What about considering `videoA` and `videoB` as deprecated, and replacing them with the new `uidA` and `uidB` in a separate `useEffect` executed on the initial page load. Then, the rest of the implementation could assume that the new uid params are used.